src/Artsy/Router/buildServerApp.tsx

```diff
@@ -107,14 +132,48 @@ export function buildServerApp(config: ServerRouterConfig): Promise<Resolve> {
             )
           }
 
+          // Build up script tags to inject into head
+          const scripts = []
+
           const { relayData: _relayData, styleTags, bodyHTML } = await trace(
             "buildServerApp.fetch",
             (async () => {
               const sheet = new ServerStyleSheet()
 
-              const html = ReactDOMServer.renderToString(
-                sheet.collectStyles(<ServerApp tags={headTags} />)
-              )
+              // Extract bundle split chunks for SSR
+              let bundleJSX
+
+              // FIXME: Remove once EXPERIMENTAL_APP_SHELL A/B test completes
+              if (getENV("EXPERIMENTAL_APP_SHELL")) {
+                const extractor = new ChunkExtractor({
+                  statsFile,
+                  entrypoints: [
+                    "ArtistApp",
+                    "ArtworkApp",
+                    "CollectApp",
+                    "CollectionApp",
+                    "CollectionsApp",
+                    "SearchApp",
```

Comment:
@zephraph - it looks like i had to explicitly set the bundles here, based on the webpack identifier strings. Maybe you can poke and see if there's a better way to automate this...

Reply:
Yeah looking at this more, this is funny -- it's loading all of these bundles up front, versus just the bundle for the page we're on. We need to play with the configuration.